include/odp/api/spec/stash_types.h

```diff
@@ -156,9 +156,38 @@ typedef struct odp_stash_capability_t {
 	 */
 	uint32_t max_stashes;
 
-	/** Maximum number of object handles per stash */
+	/** Maximum common number of object handles per stash for any object size
+	 *
+	 *  An application is able to store at least this many objects when using any of the
+	 *  supported object sizes. The per object size values listed in 'max_num' may be larger
+	 *  than this common value.
```

Comment:
"The per object size values ..." -> "Some of the per object size values..."

All values should not be larger, but some values may be larger.

Reply:
Changed in V4.